.github/ISSUE_TEMPLATE/2-feature_request.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature request
+description: Create a feature request
+title: '[Feature]: '
+labels: ['enhancement']
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time in making a Feature Request.
+
+        - This repository is maintained by volunteers, it may take a while for them to come back to you.
+        - Remember to include sufficient details and context.
+        - If you have multiple Idea's or suggestions, please submit them in separate issues.
+        - We accept pull request. If you know how to fix it - go ahead!
```

Comment:
Would be good to make it clear that it's good to make sure a PR is wanted in the first place (by talking to a maintainer or making a ticket). Not every feature is desirable at engine level